src/calculateRentalCost.js

```diff
@@ -27,8 +27,24 @@
  * @return {number}
  */
 
+const rentCarCost = 40;
+const shortTerm = 3;
+const longTerm = 7;
+const discountShortTerm = 20;
+const discountLongTerm = 50;
+
 function calculateRentalCost(days) {
-  // write code here
+  const result = days * rentCarCost;
+
+  if (days >= longTerm) {
+    return (result - discountLongTerm);
```

Comment:
You don't need parentheses here, same thing with the second return ;)
```suggestion
    return result - discountLongTerm;
```

Reply:
Oh and 'result' is not the best name, try to come up with something different ;)

Reply:
Ah! Alright, I get it. I wasn't sure, if I can live it without. :)